<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>szukajPromocji</class>
 <widget class="QDialog" name="szukajPromocji">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="szukajButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrocButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="kodCB">
      <property name="text">
       <string>Kod</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="kodLE"/>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="doProcentCB">
      <property name="text">
       <string>Do procentu zniżki</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDoubleSpinBox" name="odKwotaDSB"/>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="doKwotaCB">
      <property name="text">
       <string>Do kwoty zniżki</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="odProcentCB">
      <property name="text">
       <string>Od procentu zniżki</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="nazwaLE"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="nazwaCB">
      <property name="text">
       <string>Nazwa</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="odKwotaCB">
      <property name="text">
       <string>Od kwoty zniżki</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="doKwotaDSB"/>
    </item>
    <item row="7" column="0">
     <widget class="QSpinBox" name="odProcentZniKiSpinBox"/>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="doProcentZniKiSpinBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
